src/routes/index.js

```diff
@@ -1,2 +1,4 @@
+/* eslint-disable import/no-named-as-default */
 export AuthorizedRoute from './authorized-route'
 export UnauthorizedRoute from './unauthorized-route'
+/* eslint-disable import/no-named-as-default */
```

Comment:
Do you know of a better way to handle this? The issue is they're wrapped in an HOC when being exported so I think these names should technically be different. We could change the names, keep this single instance of disabling the rule, or we could ignore this rule globally.

Reply:
I believe this was fixed in `v6` https://github.com/LaunchPadLab/lp-components/blob/7da7efea655a0716e5988eb676be68472f1d5106/src/routes/index.js

Reply:
Ah, what's the process for that. Should I be branching off of that instead, or wait until that version is released?

Reply:
I would prefer to run formatting on v6 right before it's merged. I don't want to have to deal with the conflicts that will result from code shifting around in v5.